### ISSUE TITLE
local .env vars set to development

### ISSUE THREAD

**Issue by visualasparagus** [serverless/serverless#407]
Is there a reason that these env vars are set to `development` when you create a project? 

```
SERVERLESS_STAGE=development
SERVERLESS_DATA_MODEL_STAGE=development
```

**Comment by ryansb** [comment]
Because that's the default stage, and we expect folks to add other stages later.

**Comment by visualasparagus** [comment]
Ah. Ok. So this just provides default values to the cli interface?

**Comment by eahefnawy** [comment]
adding to @ryansb awesome answer, we're putting them as a default cause we're expecting some lambda functions might need to be aware of the stage they're running in, starting with the default "development" stage. Specially when you run the function locally.

We've also added another default env var for `SERVERLESS_PROJECT_NAME` since some lambdas might need that too. It should be out in our next release.

**Comment by visualasparagus** [comment]
It might be nice to have these values. As well as other slss defaults come from a config file located at ~/.slss

**Comment by ryansb** [comment]
Call me annoying, but I'm against dropping dotfiles right into home directories - there's modern conventions for gconf and other systems to use `~/.config/${program}/` directory. If we're going to use a config file, I'd rather have it as `~/config/serverless/conf` or `~/config/serverless/${env}.conf` if we want to have per-environment decisions. 

In that case, we'd have to read (higher in the list, higher priority)

1. environment variables
1. config for the stage (prod, dev, etc)
1. main config file

**Comment by awartani** [comment]
I hate the way environment variables are written in serverless, it should be read directly from system environment variables, and whole concept of deploying them to S3 is kind of a hack which shouldn't continue for such a great framework.

**Comment by ac360** [comment]
@awartani Environment variable handling has been completely refactored in the newly released V0.5.  More information on them can be found here: https://github.com/serverless/serverless/releases

I hope this works better for you :)

**Comment by awartani** [comment]
@ac360 that was quick, thanks for your prompt reply! great effort, yeah it looks much better now.
